anagram/tests/anagram.rs

```diff
@@ -1,11 +1,6 @@
-#![crate_name = "anagram_test"]
-#![crate_type = "lib"]
-#![feature(collections)]
-
-mod anagram;
+extern crate anagram;
 
 #[test]
-#[ignore]
```

Comment:
@lpil Some, not all.  The Ruby exercises have the first one enabled and the rest on "skip".